lib/apiservers/engine/backends/container.go

```diff
@@ -415,6 +419,44 @@ func (c *Container) ContainerRm(name string, config *types.ContainerRmConfig) er
 	return nil
 }
 
+// checkActivePortBindings gets port bindings for the container and
+// unmaps ports if the cVM that previously bound them isn't powered on
+func (c *Container) checkActivePortBindings(vc *viccontainer.VicContainer) error {
```

Comment:
`cleanupPortBindings` might be more appropriate. This name suggests no changes are made to the system.